packages/yoroi-extension/app/components/wallet/send/WalletSendForm.js

```diff
@@ -484,23 +484,19 @@ export default class WalletSendForm extends Component<Props> {
             options={sendAmountOptions}
             {...form.$('selectedAmount').bind()}
             onChange={value => {
-              this.form.$('selectedAmount').value = value
-              if(value === CUSTOM_AMOUNT && this.props.shouldSendAll) {
-                this.props.toggleSendAll();
+              if (value === CUSTOM_AMOUNT) {
+                this.props.updateSendAllStatus(false);
+                this.form.$('amount').clear();
```

Comment:
Should not clear the amount when selecting custom-amount

Reply:
Okay